docs/source/caching/normalized-cache.mdx

```diff
@@ -165,8 +165,15 @@ val response = apolloClient.query(query)
   // Always use the network and never check the cache, even if network fails
   .fetchPolicy(FetchPolicy.NetworkOnly)
 
-  // Execute the query
+  // Check the cache and also use the network (2 values can be emitted)
+  .fetchPolicy(FetchPolicy.CacheAndNetwork)
+    
+  // Execute the query (if 1 value is expected)
   .execute()
+  // Or, if several values are expected (CacheAndNetwork)
+  .toFlow().collect { response ->
+    // ...
+  }
```

Comment:
Maybe move `CacheAndNetwork` to a separate block to highlight the fact that it's a special case?

Reply:
Haha did that at first and went back and forth 😅 Ok will do 👍